src/Main.elm

```diff
@@ -11,14 +11,32 @@ import Html.Events exposing (onClick)
 
 
 type alias Model =
-    { board : List Int
-    , selectedSquare : Maybe Int
+    { selectedSquare : Maybe Int
+    , board : List BoardSquare
     }
 
 
+type alias BoardSquare =
+    { id : Int
+    , state : SquareState
+    }
+
+
+type SquareState
+    = Empty
+    | Tiled
+    | Hotelled
+
+
 init : ( Model, Cmd Msg )
 init =
-    ( { board = List.range 0 143, selectedSquare = Nothing }, Cmd.none )
+    ( { board = initBoard, selectedSquare = Nothing }, Cmd.none )
+
+
+initBoard : List BoardSquare
+initBoard =
+    List.range 0 143
+        |> List.map (\v -> { id = v, state = Empty })
```

Comment:
Initialize the board like before but now store the int as the square's id field and the state as empty